### PR TITLE
[Automation] - update resource API versions and create the Bicep file.

### PR DESCRIPTION
**Pull request by mumian**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@MGoedtel - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@MGoedtel - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@MGoedtel - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@MGoedtel - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.automation/101-automation/main.bicep [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+param automationAccountName string
+param sampleGraphicalRunbookName string = 'AzureAutomationTutorial'
+param sampleGraphicalRunbookDescription string = 'An example runbook that gets all the Resource Manager resources by using the Run As account (service principal).'
+param samplePowerShellRunbookName string = 'AzureAutomationTutorialScript'
+param samplePowerShellRunbookDescription string = 'An example runbook that gets all the Resource Manager resources by using the Run As account (service principal).'
+param samplePython2RunbookName string = 'AzureAutomationTutorialPython2'
+param samplePython2RunbookDescription string = 'An example runbook that gets all the Resource Manager resources by using the Run As account (service principal).'
+
+@description('URI to artifacts location')
+param artifactsLocation string = deployment().properties.templateLink.uri
```
Comment (bmoore-msft) [inline quickstarts/microsoft.automation/101-automation/main.bicep:35]: this needs to start with an underscore as in the original template

File: quickstarts/microsoft.automation/101-automation/main.bicep [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+@description('The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated')
+@secure()
+param artifactsLocationSasToken string = ''
```
Comment (bmoore-msft) [inline quickstarts/microsoft.automation/101-automation/main.bicep:39]: this needs to start with an underscore as in the original template

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by azure-quickstarts** [comment]
@MGoedtel - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@MGoedtel - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request closed without merging**